.github/workflows/notify_team_new_comment.yml

```
@@ -17,6 +17,12 @@ jobs:
 
     runs-on: ubuntu-latest
     steps:
+      - name: Escape title double quotes
+        id: escape_title
+        run: |
+          title=${{ github.event.issue.title }}
+          echo "ISSUE_TITLE=${title//\"/\\\"}" >> "$GITHUB_OUTPUT"
```

@MisRob @rtibbles here, we can use github's inbuilt `toJSON` function that will do the escaping for us.

Source -- https://github.com/slackapi/slack-github-action/issues/202#issuecomment-1769132156

```suggestion
          echo "ISSUE_TITLE=${{ toJSON(github.event.issue.title) }}" >> "$GITHUB_OUTPUT"
```

---

Thanks, that's nice. Initially I tried to use `toJSON` for the whole Slack payload but it seemed to be problematic, perhaps because of nested `${{ }}`? Here in separate step it seems to work well.

---

@MisRob Yes you are right, I was also going to suggest using `toJSON` on the whole payload but chances of it working right are low because of the nested `${{ }}` I think.